### PR TITLE
Remove near search for location resources.

### PR DESCRIPTION
**Pull request by LTA-Thinking**
## Description
Removes the expectation of being able to search locations based on position. The type of special is not supported yet.

## Related issues
Mitigates #603  

## Testing
All existing tests pass.

### CODE REVIEW

**Comment by hansenms** [comment]
Is there are test for this?

**Comment by LTA-Thinking** [comment]
I can write a test for adding a location, but right now there is no test.

**Comment by hansenms** [comment]
@brendankowitz , thoughts?

**Comment by brendankowitz** [comment]
Yes, lets add in an E2E create test for Location. Actually you may be able to cherry pick from here: https://github.com/microsoft/fhir-server/commit/17d49ca3ba28a9471ae24afedef6dd867f8038c9

File: src/Microsoft.Health.Fhir.R4.Core/Features/Conformance/AllCapabilities.json [hunk 1]
```diff
@@ -14802,12 +14802,6 @@
                             "type": "string",
                             "documentation": "A portion of the location's name or alias"
                         },
-                        {
```
Comment (brendankowitz) [inline src/Microsoft.Health.Fhir.R4.Core/Features/Conformance/AllCapabilities.json]: You shouldn't need to remove from this file

**Comment by LTA-Thinking** [comment]
Test added and AllCapabilities restored.

**Comment by brendankowitz** [comment]
@LTA-Thinking Looks like you might need to add to the known failures list: 
```
Failure in "searchtest_location/S000_Location", reason: "The server does not support the following params: near.", description: "Compare supported search parameters with specification Location"
```

**Pull request merged**